src/CMakeLists.txt

```diff
@@ -159,9 +159,13 @@ FILE(GLOB cc_files "${CMAKE_CURRENT_SOURCE_DIR}/[^_]*.cc")
 SET(CYCLUS_CORE_SRC ${CYCLUS_CORE_SRC}
     ${cc_files}
     "${CMAKE_CURRENT_SOURCE_DIR}/OsiCbcSolverInterface.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/transmute.c"
     )
 
-set_source_files_properties(pyne_decay.cc PROPERTIES COMPILE_FLAGS "-O0")
+add_definitions(-DPYNE_DECAY_IS_DUMMY)
```

Comment:
It looks like you're removing the file that references this, do you still need it?